src/declaration.d

```diff
@@ -128,12 +128,16 @@ enum STCautoref             = (1L << 45);   // Mark for the already deduced 'aut
 enum STCinference           = (1L << 46);   // do attribute inference
 enum STCexptemp             = (1L << 47);   // temporary variable that has lifetime restricted to an expression
 enum STCmaybescope          = (1L << 48);   // parameter might be 'scope'
+enum STCfuture              = (1L << 49);   // introducing new base class function
```

Comment:
I don't see a use for it in general, because other symbols would just get overridden automatically except those at public module scope.